levels.js

```diff
@@ -104,6 +105,7 @@ var levelOne = {
 };
 
 var bonusLevelOne = {
+    label: "1-1",
```

Comment:
shouldn't this be labeled something slightly differently to differentiate it? I assume this new "label" field is to be used by the new level selection mechanism to select the object with the specified label.